scripts/fbp-runner.sh

```diff
@@ -19,21 +19,22 @@ SCRIPT="$3"
 ENV_PATH="$2"
 SERVICE="fbp-runner@"$(systemd-escape $ENV_PATH)
 
-systemctl stop $SERVICE
+if [ $1 == "stop" ]; then
+	systemctl stop $SERVICE
 
-if [ $1 == "start" ]; then
+elif [ $1 == "start" ]; then
```

Comment:
You are right I will add restart this will keep the old behavior and avoid asking password twice, I will make a V2